public/app/emotion.ts

```diff
@@ -0,0 +1,5 @@
+import createEmotion from '@emotion/css/create-instance';
```

Comment:
Hmm... we share `@emotion/css` with plugins via sharedDependencies.ts - how do plugins that use emotion behave now?